core/src/main/java/org/apache/accumulo/core/clientImpl/bulk/BulkImport.java

```diff
@@ -302,35 +302,25 @@ public MLong(long i) {
   }
 
   public interface KeyExtentCache {
-    KeyExtent lookup(Text row)
-        throws AccumuloException, AccumuloSecurityException, TableNotFoundException;
+    KeyExtent lookup(Text row);
   }
 
   public static List<KeyExtent> findOverlappingTablets(KeyExtentCache extentCache,
-      FileSKVIterator reader)
-      throws IOException, AccumuloException, AccumuloSecurityException, TableNotFoundException {
-
-    Text startRow = null;
-    Text endRow = null;
+      FileSKVIterator reader) throws IOException {
 
     List<KeyExtent> result = new ArrayList<>();
     Collection<ByteSequence> columnFamilies = Collections.emptyList();
-    Text row = startRow;
-    if (row == null)
-      row = new Text();
+    Text row = new Text();
     while (true) {
-      // log.debug(filename + " Seeking to row " + row);
       reader.seek(new Range(row, null), columnFamilies, false);
       if (!reader.hasTop()) {
-        // log.debug(filename + " not found");
         break;
       }
       row = reader.getTopKey().getRow();
       KeyExtent extent = extentCache.lookup(row);
-      // log.debug(filename + " found row " + row + " at location " + tabletLocation);
       result.add(extent);
       row = extent.endRow();
-      if (row != null && (endRow == null || row.compareTo(endRow) < 0)) {
```

Comment:
I can't tell what this was trying to do, but clearly your change is equivalent to existing behavior if `endRow` was never assigned to anything other than `null`.

Reply:
I think the method originally took `startRow` and `endRow` as parameters but an update made that not needed.  This [update](https://github.com/apache/accumulo/commit/7ef140ec40c3768859b848350db8c6d6d20f7a56#diff-f36575975d4075ddd82d4963f08430c2ee6755baed4949b4e96e82b8eb7810baR266) changed the parameters.